src/ui/components/results/alternativeverticalscomponent.js

```diff
@@ -96,9 +107,13 @@ export default class AlternativeVerticalsComponent extends Component {
    * @param {object} alternativeVerticals alternativeVerticals server response
    * @param {object} verticalsConfig the configuration to use
    */
-  static _buildVerticalSuggestions (alternativeVerticals, verticalsConfig) {
+  static _buildVerticalSuggestions (alternativeVerticals, verticalsConfig, context) {
     let verticals = [];
-    let queryParams = window.location.search;
+
+    const params = {};
+    if (context) {
+      params[StorageKeys.API_CONTEXT] = context;
+    }
```

Comment:
What about the other params in window.location.search? Seems like we were including those in the generated url before. Does add paramsToUrl take care of that for us?

Reply:
Yup!
```
export function addParamsToUrl (url, params = {}) {
  const urlParams = new SearchParams(window.location.search.substring(1));
```